### ISSUE TITLE
Pytorch Learning Rate Scheduler and fp16 optimizer

### ISSUE THREAD

**Issue by arijitthegame** [NVIDIA/apex#1197]
HI, 

 What is the recommended way to use fp_16 optimizer and lr rate scheduler? 

 For example : If I do 
'''
optimizer = apex.fp16_utils.FP16_Optimizer(optimizer, dynamic_loss_scale=True)
lr_scheduler = torch.optim.lr_scheduler.LambdaLR(
    optimizer=optimizer, **kwargs)
'''

However that throws an error as this wrapped optimizer is not an instance of PyTorch optimizer. So what is the best way to use a scheduler along with a fp_16 optimizer?
